modules/apps/announcements/announcements-web/src/main/resources/META-INF/resources/announcements/configuration.jsp

```diff
@@ -100,20 +100,18 @@ announcementsPortletInstanceConfiguration = ParameterMapUtil.setParameterMap(Ann
 
 								<%
 								List<KeyValuePair> leftList = new ArrayList<KeyValuePair>();
-
-								for (Group curGroup : groups) {
-									if (announcementsDisplayContext.isScopeGroupSelected(curGroup)) {
-										leftList.add(new KeyValuePair(String.valueOf(curGroup.getGroupId()), curGroup.getDescriptiveName(locale)));
-									}
-								}
-
 								List<KeyValuePair> rightList = new ArrayList<KeyValuePair>();
 
 								for (Group curGroup : groups) {
-									KeyValuePair tempKeyValuePair = new KeyValuePair(String.valueOf(curGroup.getGroupId()), curGroup.getDescriptiveName(locale));
+									String descriptiveName = curGroup.isOrganization() ? String.format("%s(%s)", curGroup.getDescriptiveName(locale), LanguageUtil.get(request, OrganizationConstants.TYPE_ORGANIZATION)) : curGroup.getDescriptiveName(locale);
```

Comment:
A minor aesthetic issue: shouldn't the format string be `"%s (%s)"` (with a blank before the paren)?

Other than that, i looks good. If that's on purpose (e.g. getDescriptiveName contains trailling blanks), we can forward this to Brian.

Reply:
Hi @adolfopa, there was no reason for the lack of space.  I've updated the pull to include it, and force-pushed the changes now.

Please let me know if you have any additional questions, thanks!